bot_api_py/events/skipped_turn_event.py

```diff
@@ -0,0 +1,14 @@
+from bot_api_py.events.bot_event import BotEvent
+
+
+class SkippedTurnEvent(BotEvent):
```

Comment:
This class lacks the SkippedTurnEvent(int turnNumber) { super(turnNumber); } (init method) as the user should be able to read out the turn_number, which is crucial to skipped turns.

Reply:
Same as above.